openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -526,6 +526,14 @@ def _get_orgs_for_user(user):
             # Org can concievably be blank or this bogus default value
             if org and org != 'outdated_entry':
                 orgs.add(org)
+        try:
+            # if the user has ever launched a managed Zoom xblock,
+            # we'll notify Zoom to delete their records.
+            if user.launchlog_set.filter(managed=True).count():
```

Comment:
Just double-checking but my mental model of this is that there should only ever be 1 or 0 launchLogs per user, right? In that case I'd find this more readable if we used `get` and caught/handled the multiple objects returned versus the no object found cases separately, with some logging in the unexpected cases

Reply:
There could be many launchlogs. One per course.

Reply:
Then we should probably be using SQL EXISTS, rather than counting them and seeing if the total is nonzero